swarm/pss/pss_test.go

```diff
@@ -197,10 +209,23 @@ func TestCache(t *testing.T) {
 		t.Fatalf("message %v should NOT have EXPIRE record in cache but checkCache returned true", msgtwo)
 	}
 
-	time.Sleep(pp.CacheTTL)
+	time.Sleep(pp.CacheTTL + 1*time.Second)
```

Comment:
pp.CacheTTL should be set explicitly here so we know we're testing the right interval. Also, we don't really need to waste a whole second ;)

Reply:
I had an issue earlier:
* message was added to the queue
* expiration check ticks (during `time.Sleep(pp.CacheTTL)` but the message had not yet expired)
 * sleep finishes and the message had not yet been removed.  Test condition is executed before the next tick of the expiration check.

So here, would need to wait (at max), two ticks of the cache expiration check to ensure that a message was removed, hence the additional time.

But this is now passing if I change `time.Sleep(pp.CacheTTL + 1*time.Second)` to `time.Sleep(pp.CacheTTL)`.  I can revert this line but I'm unsure whether we will see this case fail because of the conditions I described above.

Reply:
Oh, I'm sorry, I misread the first time. You have the ps.CacheTTL already in there, of course %) fine!

This is total nitpick, but: Maybe just explicitly set ps.CacheTTL to Millisecond, and sleep 2*ps.CacheTTL? That should do it, no?